tests/bud.bats

```diff
@@ -43,6 +43,13 @@ load helpers
   expect_output --substring "unsupported value"
 }
 
+@test "build with inline RUN --network=default" {
+  run_buildah build --network=host $WITH_POLICY_JSON -t source -f $BUDFILES/inline-network/Dockerfile4
+  build1="$output"
+  run_buildah build --network=private $WITH_POLICY_JSON -t source -f $BUDFILES/inline-network/Dockerfile4
+  assert "$output" !~ "$build1"
```

Comment:
... and running with `--network=private` should get a result that's both different from the `--network=host` test and the previous `--network=private`, unless I'm mistaken.

Reply:
Did you mean different with the previous `--network=host` and another `--network=private` ? Yes I think two different builds should have different namespaces. 

I can add two builds with `--network=private` and compare them if you mean that.

Reply:
Yes, that's what I was suggesting, since we're creating a new network namespace each time.

Reply:
This `assert` probably needs to be changed - it's comparing the entire output from the previous build to the entire output of this one.  Since we're not caching, I'd expect to get different image IDs in those outputs, at least.

Reply:
It seems buildah re-uses private namespace and only assigns new private namespace when other one is occupied, so this happens only when two private builds are parallel otherwise consecutive private build ends up getting same private namespace.

Reply:
That's surprising to me, but I'm seeing it when I run `unshare -Urnmpf readlink /proc/self/ns/net` twice in succession, so the takeaway is that it's something the kernel's doing, and I was wrong to think that they'd be different.

Reply:
ah i see its the kernel, anyways I have dropped this part from the test, only comparing a `host` and `private` ns build.